### DISCUSSION TITLE
Workflow on pull request doesn't show the last result

### DISCUSSION THREAD

**Discussion by kaojohnny28** [community/community#25574] [Actions]
<p>I have a workflow which linting pull request title, format, and it is triggered when:</p>
<pre><code>[opened, synchronize, edited, reopened]
</code></pre>
<p>When a pull request is updated, the workflow is triggered correctly, but having the following problems:</p>
<ol>
<li>
<p>In the pull request merge panel, it shows multiple workflow trigger history, instead of the last one.</p>
</li>
<li>
<p>In the pull request listing page, it always shows failed (red corss icon) even the pull request is passed afterward.</p>
</li>
</ol>
<p>This behavior is not what I expect, may I know there is any way to adjust the result?</p>
<p>Thanks.</p>

**Comment by oldskool** [comment]
<blockquote>
<p><a class="mention" href="/kaojohnny28">@kaojohnny28</a> wrote:</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<ol>
<li>In the pull request merge panel, it shows multiple workflow trigger history, instead of the last one.</li>
</ol>
</blockquote>
<p>This would be the expected behaviour. Each trigger is logged and shown. It’s not that one trigger overwrites the other. You never want to lose sight of the history that happened on your workflow, otherwise debugging/testing would become very difficult. Plus if it would just start “hiding” things from you by overwriting previous runs, it would also complicate transparency when it comes to billing (where applicable). You would never be able to see where your build minutes would come from.</p>
<blockquote>
<p><a class="mention" href="/kaojohnny28">@kaojohnny28</a> wrote:</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<ol start="2">
<li>In the pull request listing page, it always shows failed (red corss icon) even the pull request is passed afterward.</li>
</ol>
</blockquote>
<p>I don’t quite understand what you mean here. This would generally mean that your workflow failed, but without seeing any of your workflow or the output it generates, it’s hard to say. I’ve never seen this behaviour in any of my workflows before, so I don’t think this is a common bug or something that would be unexpected unless something actually goes wrong.</p>

**Comment by kaojohnny28** [comment]
<p>Hi <a class="mention" href="/oldskool">@oldskool</a> thanks for your reply, for point 1, I can understand. For point 2, the pull request listing pages, I mean</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0be40530cb5cb04ce75fa7ba2370c1d7f2109aff.png" data-download-href="/uploads/short-url/1HbMzfh7VgphO7HzgKFWuZpOVC7.png?dl=1" title="Screenshot_2019_11_19__10_08_AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/50310716/181091867-e9bdebea-50d1-4a20-b183-905849301f66.png" alt="Screenshot_2019_11_19__10_08_AM.png" data-base62-sha1="1HbMzfh7VgphO7HzgKFWuZpOVC7" width="690" height="131" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0be40530cb5cb04ce75fa7ba2370c1d7f2109aff_2_690x131.png, https://aws1.discourse-cdn.com/github/original/2X/0/0be40530cb5cb04ce75fa7ba2370c1d7f2109aff.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0be40530cb5cb04ce75fa7ba2370c1d7f2109aff.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0be40530cb5cb04ce75fa7ba2370c1d7f2109aff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2019_11_19__10_08_AM.png</span><span class="informations">1016×193 17.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The red corss icon means the workflow failed, but it doesn’t change to a green check icon once the workflow runs successfully. It’s my idea that here it should show the latest status rather than fail always. Do you think it is possible?</p>
<p>Thanks.</p>

**Comment by oldskool** [comment, marked as answer]
<blockquote>
<p><a class="mention" href="/kaojohnny28">@kaojohnny28</a> wrote:</p>
</blockquote>
<blockquote>
<p>The red corss icon means the workflow failed, but it doesn’t change to a green check icon once the workflow runs successfully.</p>
</blockquote>
<p>That is what should happen and what does happen for me. I’ve never seen it “stick” to a status after it changed. Maybe it’s a caching issue, but if you persistently see this and (hard) refreshing doesn’t help I think you’d best contact GitHub support about this.</p>
